app/domain/authentication/authn_jwt/signing_key/create_signing_key_interface.rb

```diff
@@ -0,0 +1,75 @@
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # Factory that returns the interface implementation of FetchSigningKey
+      CreateSigningKeyInterface ||= CommandClass.new(
+        dependencies: {
+          fetch_provider_uri: Authentication::AuthnJwt::SigningKey::FetchProviderUriSigningKey,
+          fetch_jwks_uri: Authentication::AuthnJwt::SigningKey::FetchJwksUriSigningKey,
```

Comment:
can we add `.new` to these 2 so they are created once?

Reply:
No because they get parameters in the constructors

Reply:
in that case then these parameters should be set as inputs and not dependencies. then you can add `.new` and pass the params in the `.call` method.